<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nightwood - Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="topMargin">
    <number>60</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>60</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 40px; font-weight: bold; color: #2d3436; letter-spacing: 1.2px; padding-bottom: 10px;</string>
     </property>
     <property name="text">
      <string>Nightwood</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="minimumHeight">
      <number>55</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subtitleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 14px; color: #636e72; margin-top: 0px; margin-bottom: 10px;</string>
     </property>
     <property name="text">
      <string>Create a new account to get started</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="fullNameEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  padding-left: 15px;
  font-size: 15px;
  border-radius: 10px;
  background-color: white;
  border: 1px solid #dfe6e9;
  color: #2d3436;
}
QLineEdit:focus {
  border: 2px solid #6c5ce7;
}</string>
     </property>
     <property name="placeholderText">
      <string>Full Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  padding-left: 15px;
  font-size: 15px;
  border-radius: 10px;
  background-color: white;
  border: 1px solid #dfe6e9;
  color: #2d3436;
}
QLineEdit:focus {
  border: 2px solid #6c5ce7;
}</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passphraseEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  padding-left: 15px;
  font-size: 15px;
  border-radius: 10px;
  background-color: white;
  border: 1px solid #dfe6e9;
  color: #2d3436;
}
QLineEdit:focus {
  border: 2px solid #6c5ce7;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Passphrase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confirmPassphraseEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
  padding-left: 15px;
  font-size: 15px;
  border-radius: 10px;
  background-color: white;
  border: 1px solid #dfe6e9;
  color: #2d3436;
}
QLineEdit:focus {
  border: 2px solid #6c5ce7;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm Passphrase</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="requirementsLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="togglePassphraseButton">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  border: none;
  background-color: transparent;
  padding: 0px;
  font-size: 13px;
  color: #6c5ce7;
  font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="passphraseRequirementsLabel">
       <property name="styleSheet">
        <string notr="true">color: #636e72; font-size: 10px; margin-left: 5px; margin-top: 1px;</string>
       </property>
       <property name="text">
        <string>Passphrase must be at least 15 characters long</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="registerButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  font-size: 16px;
  font-weight: bold;
  color: white;
  background-color: #6c5ce7;
  border-radius: 10px;
  border: none;
}
QPushButton:hover {
  background-color: #5049c9;
}
QPushButton:pressed {
  background-color: #4040b0;
}</string>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QFrame" name="line1">
       <property name="styleSheet">
        <string notr="true">background-color: #dfe6e9;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="orLabel">
       <property name="styleSheet">
        <string notr="true">color: #636e72; font-size: 13px;</string>
       </property>
       <property name="text">
        <string>or</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="line2">
       <property name="styleSheet">
        <string notr="true">background-color: #dfe6e9;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  font-size: 16px;
  font-weight: bold;
  border-radius: 10px;
  color: #6c5ce7;
  background-color: white;
  border: 2px solid #6c5ce7;
}
QPushButton:hover {
  background-color: #f5f3ff;
}
QPushButton:pressed {
  background-color: #eeeaff;
}</string>
     </property>
     <property name="text">
      <string>Already have an account? Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="passphraseWarningLabel">
     <property name="styleSheet">
      <string notr="true">color:rgb(198, 36, 18); font-size: 10px; margin-left: 5px; margin-top: 3px; font-weight: bold; letter-spacing: 0.3px;</string>
     </property>
     <property name="text">
      <string>Ensure you remember your passphrase
It cannot be recovered</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
